package.json

```diff
@@ -18,7 +18,7 @@
   "scripts": {
     "build": "mkdocs build --clean",
     "markdownlint": "markdownlint-cli2 \"**/*.md\" \"!**/node_modules/**\"",
-    "linkinator": "linkinator site --recurse --silent --skip \"^(?!http://localhost)\"",
+    "linkinator": "npm run build && linkinator site --recurse --silent --skip \"^(?!http://localhost)\"",
```

Comment:
Without this change, `npm test` does not run properly unless you run `npm run build && npm test` explicitly.